README.md

```diff
@@ -0,0 +1,74 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Weather App
+
+> This is a weather app that allows you to search for a city and get the current weather information,
+Get all the information you need to plan your day. The app displays the current temperature, the weather description, the humidity, the wind speed, and the pressure.
+
+<br>
+<br>
+
+## Built With
+
+<br>
+<img alt="React Badge" src="https://img.shields.io/badge/React-20232A?style=for-the-badge&logo=react&logoColor=61DAFB">
+<img alt="Redux" src="https://img.shields.io/badge/Redux-593D88?style=for-the-badge&logo=redux&logoColor=white">
+<img alt="HTML Badge" src="https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white">
+<img alt="CSS Badge" src="https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white">
+<img alt="JavaScript Badge" src="https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E">
+<img alt="Jest Badge" src="https://img.shields.io/badge/Jest-C21325?style=for-the-badge&logo=jest&logoColor=white"><br>
+
+<br>
+<br>
+
+## Getting Started with Create React App and Redux
+
+<br>
+
+To get a local copy up and running follow these simple example steps:
+
+- Clone this repo to your computer using git.
+Or
+- Download the compressed [zip folder](https://github.com/tiagomarin/Space-Travelers-Hub/archive/refs/heads/development.zip) and use a decompressor to unzip it into your computer. Then open the folder and all the project files will be there.
+
+- then instal npm with "npm install" in your terminal and then **use "npm start"** to open the application on your browser or **use "npm test"** to check the available test.
+
+<br>
+<br>
+
+## Authors
+
+👤 **Andy Menutti**
+
+- GitHub: [@andym80](https://github.com/andym80)
+- LinkedIn: [LinkedIn](http://lnnk.in/ekew)
+
+<br>
+<br>
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<br>
+<br>
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+<br>
+<br>
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
```

Comment:
- [ ] Nice working on your Readme, kindly add appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). should